### DISCUSSION TITLE
"CUDA_ERROR_OUT_OF_MEMORY" issue

### DISCUSSION THREAD

**Discussion by phragmosis** [talmolab/sleap#1745] [Help!]
Hi all,

I'm trying to track multiple large videos on v 1.3.3. (each video has ~85K frames at 2.7K resolution), and I get to around 2500 frames before it stops and I get the error to check my terminal for more info, where it seems to suggest it is out of memory (please see attached txt file for full output).

This has happened on three different Windows machines now, including one where different but identically-sized videos used to work on a previous version of SLEAP (v 1.3.0), which should have plenty of GPU. I am certainly not well versed in Python, but have tried the other suggestions of restarting my computer etc, to no avail.

Any help would be massively appreciated!

Cheers,
Rob

[Memory error.txt](https://github.com/talmolab/sleap/files/15010027/Memory.error.txt)

**Comment by Marielamend** [comment]
Hello @phragmosis!

It seems like this issue might stem from this error.
```
        raise ValueError("Found multiple nodes named ({}).".format(name))
ValueError: Found multiple nodes named (name).
```
If you can, do you mind sharing your slp project file [here](https://docs.google.com/forms/d/e/1FAIpQLSfWBwcuZH8lxGyFabMAODLjSUE4YrSCYbrrnYvXCpgBqZpcJA/viewform) so we can further assess!

Best, 
Mariela

**Reply by phragmosis** [reply]
Amazing, thank you Mariela! I've uploaded the slp file, along with the models and an example video, to the link. Thank you very much for looking into this so quickly!

Cheers,
Rob

**Reply by Marielamend** [reply]
Perfect! Thank you for uploading all that, we will look into it!

Best, 
Mariela

**Reply by Marielamend** [reply]
Hello @phragmosis!

Sorry for the delay!
I went ahead and opened up your project slp + video from my end, but it seems that the slp file you sent does not have any labels or a skeleton. If possible, can you send the project slp file that you used when you got this error above. 

From the file you had provided above, it seems the slp file used was: `Danio_albolineatus_dropbox_labels.v004.slp`
If you can provide this specific slp file via the [same form again](https://docs.google.com/forms/d/e/1FAIpQLSfWBwcuZH8lxGyFabMAODLjSUE4YrSCYbrrnYvXCpgBqZpcJA/viewform), that would be great!

Have a great day, 
Mariela
